modular_zzplurt/code/modules/cargo/packs/costumes_toys.dm

```diff
@@ -0,0 +1,11 @@
+// Modifies Blastwave costumes to use newer ported (goner) ones.
+/datum/supply_pack/costumes_toys/blastwave
+	cost = CARGO_CRATE_VALUE * 2.5
+	contains = list(
+		/obj/item/clothing/under/goner/fake = 1,
+		/obj/item/clothing/suit/goner/fake = 1,
+		/obj/item/clothing/head/helmet/goner/fake = 1,
+		/obj/item/clothing/head/helmet/goner/officer/fake = 1,
+		/obj/item/clothing/mask/gas/goner/fake = 1,
+		/obj/item/clothing/shoes/jackboots = 1
+	)
```

Comment:
what about the original objects of this crate tho?

Reply:
The original crate essentially contained the same but older (upstream). I admit I made it cheaper, however.